packages/css/src/spinner/spinner.css

```
@@ -0,0 +1,29 @@
+@keyframes hds-spinner-spin {
```

Burde eksplisitt importere `_icons.css`, da vil det fortsatt fungere om konsumenten bare importerer `spinner.css` isteden for `index.css`. Slik som i [accordion](https://github.com/bring/hedwig-design-system/blob/master/packages/css/src/accordion/accordion.css#L2)

+ Da kan vi fjerne fjerne stylelint disable nedenfor.
```
  /* stylelint-disable-next-line csstools/value-no-unknown-custom-properties -- Custom icon` */
```

```suggestion
@import url("../_icons.css");

@keyframes hds-spinner-spin {
```

---

Nice! Da skal jeg fikse det!